File: Data Structures & Algorithms/concatenation-of-array/submission-2.py
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        n = len(nums)
        rnums = [0]*(n*2)
        for i in range(len(rnums)):
            if i<len(nums):
                rnums[i] = nums[i]
                continue
            rnums[i] = nums[i-n]
        return rnums

File: Data Structures & Algorithms/two-integer-sum/submission-2.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        int x;
        for(int i=0; i<nums.size(); i++) {
            x=target-nums[i];
            for(int l=i+1; l<nums.size(); l++) {
                if(nums[l]==x) {
                    return {i,l};
                }
            }
        }
        return {0,1};
    }
};
